oggm/tests/test_prepro.py

```diff
@@ -515,9 +515,20 @@ def test_gridded_data_var_to_geotiff(self):
         with xr.open_dataset(gdir.get_filepath('gridded_data')) as ds:
             gridded_topo = ds[target_var]
             gtiff_ds = salem.open_xr_dataset(gtiff_path)
-            assert ds.salem.grid == gtiff_ds.salem.grid
+            # this line is uncommented due to a bug in salem, maybe also change
+            # it to check coordinates like below (floating point issues)
+            # assert ds.salem.grid == gtiff_ds.salem.grid
             assert np.allclose(gridded_topo.data, gtiff_ds.data)
 
+        # compare coordinates of topo.tif with dem.tif
+        demtiff_ds = salem.open_xr_dataset(gdir.get_filepath('dem'))
```

Comment:
Can we try to avoid using salem here, but use rioxarray instead? It might be possible to use rioxarray to open but still use salem's accessor.

Reply:
Yes, I can use rioxarray to open the tiff files and compare the coordinates, I will change this. But I can not use salem's accessor because their is no variable `.pyproj_srs` added to the dataset when opening with rioxarray.

Reply:
OK yes I hoped that the coords would be enough, but indeed the proj info is missing - no worries, that's enough as is for now.